app/assets/v2/js/pages/new_bounty.js

```diff
@@ -25,19 +26,21 @@ $(document).ready(function(){
     if(localStorage['bountyType']){
         $('select[name=bountyType] option:contains('+localStorage['bountyType']+')').prop('selected', true);
     }
-    if(localStorage['issueURL']){
-        $('input[name=issueURL]').val(localStorage['issueURL']);
-    }
+    // if(localStorage['issueURL']){
```

Comment:
should this be commented out?

Reply:
So I commented that guy out because what tends to happen sometimes (granted this might be a development/testing problem only), is that when you refresh the page, it loads in the github URL into the field since it is cached.

However -- The scraping of the github issue does not actually happen unless you click on and off that box.  So when I was testing I was getting a lot of bounties without titles or descriptions because I forgot to click on and off the box.  Not caching the github URL was a fix for that.  Maybe not the best fix but it forces the action.

Reply:
ah okk.... i belive that somewhere down the page i had some code that would auto-retrieve the github issue title/keywords IFF this field was empty upon load.

maybe this isnt working on this branch?? im not sure.. i think we might need to merge the latest master in

Reply:
I'll merge in the latest.  Need to fix the merge conflicts anyway.

Reply:
id love to get this uncommented out and then reissue the `fetchTitle()` and related calls IFF `(localStorage['issueURL']` is not falsy

Reply:
ok i will update this

Reply:
Gotcha, yes I'm bringing the localStorage back.  Will make sure its working properly with the scrape and all.